### PR TITLE
Use commit SHAs for generating CDN-backed URLs of charts, fix #5

### LINKED ISSUE #5: Prevent CDN caching images at `master` URLs
Steps to reproduce:
1. View a chart on https://echeung.me/uoft-enrolment-charts/# (eg., so that the page loads https://cdn.rawgit.com/arkon/uoft-enrolment-charts/master/charts/CSC/CSC358H1S.svg)
1. Update the chart
2. Attempt to view the updated chart via the website

Expected: new chart appears
Observed: new chart does not appear until the previous version expires off the CDN

**What can be done:**
We should use the latest SHA of `master` reference instead of directly referencing the branch name in the chart URL (eg., https://cdn.rawgit.com/arkon/uoft-enrolment-charts/dd195f3b3626f5b7a66aa200fabaa83fdb13813c/charts/CSC/CSC358H1S.svg).

We already fetch the repo tree [here](https://github.com/arkon/uoft-enrolment-charts/blob/master/site/src/assets/js/script.js#L63); since the call returns the latest SHA under `sha` key, we could just use it to form the appropriate chart URL.

### PR DESCRIPTION
**Pull request by freeatnet**
Fixes #5

### CODE REVIEW

File: site/src/assets/js/script.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ var getCharts = function(data) {
     } else if (chart.type === 'blob') {
       $elCourse = $('<li>').text(getCourseFromPath(chart.path));
       $elCourse.addClass('course');
-      $elCourse.attr('data-chart', chart.path);
+      $elCourse.attr('data-chart', data.sha + "/" + chart.path);
```
Comment (arkon) [inline site/src/assets/js/script.js]: Could we use single quotes?

Comment (freeatnet) [inline site/src/assets/js/script.js]: Done

**Comment by freeatnet** [comment]
@arkon Could you deploy it whenever you have a moment?

**Comment by arkon** [comment]
Done!

**Pull request merged**